packages/ui-components/src/components/range/readme.md

```diff
@@ -45,12 +45,20 @@ export const KvRangeExample: React.FC = () => (
 | Name                         | Description                        |
 | ---------------------------- | ---------------------------------- |
 | `--range-height`             | Height of the range slider         |
+| `--range-height`             | Height of the range slider         |
```

Comment:
There are some repeated props on the readme files

Reply:
You're right, will revert this changes as well 👍